Added VMware ESXi

DanAE111 opened the pull request:
Addition of EOL dates for VMware ESXi hypervisor.

DanAE111 committed:
Added VMware ESXi

Addition of EOL dates for VMware ESXi hypervisor.

DanAE111 committed:
Rename esxi to esxi.md

doh!

captn3m0 committed:
[esxi] Adds command and removes comments

captn3m0 committed:
[esxi] Fixes link

usta reviewed (changes requested):
How can a product's support date is longer than its eol date ?
@captn3m0  @DanAE111

DanAE111 committed:
Update esxi.md

Elaborated on product lifecycle cadence and swapped eol and support dates making both visible.
